alphagov/paas-cf
# Increase disk size of large postgres plans

What
----

Per https://github.com/alphagov/paas-team-manual/pull/415

How to review
-------------

Try it out in a dev env:
* Try upgrading from a medium high-iops plan to a old large plan, expect failure.
* Try upgrading from a medium high-iops plan to a new large plan, expect success.
* Try upgrading from an old large plan to a new large plan, expect success.
* Try creating an instance with the old plan, expect failure.

---

🚨⚠️ Please do not merge this pull request via the GitHub UI ⚠️🚨
